Adding warning to Warehouse Selective Sync section (#4772)
segmentio/segment-docs: src/connections/storage/warehouses/warehouse-syncs.md

> warning ""
> For each warehouse only the first 5,000 collections per source and 5,000 properties per collection are visible in the Selective Sync user interface. [Learn more about the limits](#selective-sync-user-interface-limits).

> > warning ""

Reviewer: ```suggestion
> error "Deleting the `received_at` column may cause permanent data loss"
```

> > All tables use `receieved_at` for the sort key, disabling this column will cause your syncs to fail.

Reviewer: ```suggestion
> Disabling the `recieved_at` column will cause your syncs to fail, as all tables use `receieved_at` as the sort key.
```
